kb/development/script-coding-standards.md

```diff
@@ -0,0 +1,102 @@
+# Mbed TLS script coding standards
+
+## Choice of scripting language
+
+The preferred language for scripts in the Mbed TLS project is Python. Use Python for new scripts that are committed to the project unless there's a good reason.
+
+Some reasons not to use Python:
+
+* A script that only applies to Windows and should ideally work out of the box can be written in cmd.
+* Some CI code in the `mbedtls-test` repository is written in Groovy, for its Jenkins integration.
+* A script only intended for the Mbed TLS developers, and whose job is mostly to run other programs, can be written in sh.
+* If you're extending existing scripts, then obviously keep using the same language.
+* Throwaway scripts that do ad hoc automated refactoring (e.g. renaming identifiers) are often in Perl, thanks to its convenient text manipulation primitives.
+
+### Language versions
```

Comment:
So the file is "script-coding-standards", and it (very reasonably) starts with a section on choice of language, and (very reasonably) has a section on python coding standards, but in between there are sections on language version, which is only about python, and naming conventions, which is about all types of scripts, and invocation, which seems to only be about Unix systems (do we expect scripts written for `cmd.exe` to take `--help` or `/help`?

Suggestion: move the "Language versions" section into the Python-specific part of this document